src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -557,18 +585,32 @@ void setSctpConnection(SctpConnection sctpConnection)
 
             if (sctpConnection != null)
             {
-                if (sctpConnection.isReady())
-                {
-                    notifyTransportChannelConnected();
-                }
-                else
-                {
-                    sctpConnection.addChannelListener(webRtcDataStreamListener);
-                }
+                hookUpDefaultWebRtcDataChannel(sctpConnection);
```

Comment:
Why do we no longer need to check if the sctpConnection is ready?

Reply:
The behavior of `SctpConnection.getDefaultDataStream()` called from `hookUpDefaultWebRtcDataChannel` has changed to only select the stream and return `null` if there isn't any. Previously this method would try to open a new one which involves writing on the sctp socket and for that it had to be ready.